docs/mkdocs.yml

```diff
@@ -15,11 +15,16 @@ theme:
         - navigation.tabs
         - navigation.instant  # Load pages via XMLHttpRequest (XHR)
         - navigation.tracking
+    palette:
+        primary: white
```

Comment:
@hoechenberger FYI these are the two lines we'll need to revert if the theme-toward-pydata is too much :)

I'll actually merge as-is as we can always fix it easily afterward, I don't think it'll matter for cutting the release